src/components/Navbar.tsx

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import {
+  AppBar,
+  Toolbar,
+  Button,
+  Typography,
+  IconButton,
+} from "@material-ui/core";
+import MenuIcon from "@material-ui/icons/Menu";
+import {
+  Drawer,
+  ListItem,
+  List,
+  ListItemIcon,
+  ListItemText,
+} from "@material-ui/core/";
+import { useState } from "react";
+import { NavLink } from "react-router-dom";
+import DropDownMenu from "./DropDownMenu";
+import { makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles({
+  root: {
+    fontFamily: "Open Sans, sans-serif",
+  },
+  logo: {
+    color: "black",
+    fontSize: "30px",
+    marginBottom: "10px",
+  },
+  title: {
+    flexGrow: 1,
+    marginLeft: "10px",
+    color: "grey",
+  },
+  toolbar: {
+    backgroundColor: "white",
+  },
+  header: {
+    boxShadow: "0px 0px 0px 0px",
+  },
+  list: {
+    width: 250,
+  },
+  btnpost: {
+    border: "1px solid grey",
+    borderRadius: "20px",
+  },
+});
+
+const Navbar: React.FC = () => {
+  const classes = useStyles();
+  const [open, setOpenDrawer] = useState(false);
+  const handleDrawer = () => {
+    setOpenDrawer(true);
+  };
+  return (
+    <>
+      <AppBar className={classes.header}>
+        <Toolbar className={classes.toolbar}>
+          <NavLink to="/" className={classes.logo}>
+            jobshop.it
+          </NavLink>
+          <Typography className={classes.title}>
+            #1 Job Board for IT industry in Poland
```

Comment:
Jak już wrzucisz ten plik do osobnego folderu navbar -> navbar.tsx. To zrób w tym folderze (navbar) folder consts -> i tutaj zrób plik navigationPages.const.ts -> i w tym pliku trzymaj wszystkie stringi

Reply:
Jak już będziesz miała wszystkie stringi w pliku navigationPages.const.ts -> to użyj ich tutaj. Czyli np w pliku navigationPages.const.ts robisz sobie:
`const pageTitleText = '#1 Job Board for IT industry in Poland'`
i w navbar.tsx używasz po prostu:
`<Typography className={classes.title}>{pageTitleText}</Typography>`